log/logger.go

```diff
@@ -18,20 +18,41 @@
 package log
 
 import (
+	"context"
+	"fmt"
+	"github.com/google/uuid"
+	"github.com/pkg/errors"
+	"golang.org/x/sys/unix"
 	"io"
 	"os"
+	"strings"
 	"sync"
 	"time"
 
 	"go.uber.org/zap"
 	"go.uber.org/zap/zapcore"
+	"gopkg.in/natefinch/lumberjack.v2"
 )
 
+const ObkvTraceId = "_ObkvTraceIdName"
```

Comment:
I mean rename "ObkvTraceId" to “ObkvTraceIdName”，"ObkvTraceId"  look like a integer variable

Reply:
down